src/models/service/transactions/details.rs

```diff
@@ -30,6 +31,19 @@ pub struct MultisigExecutionDetails {
     pub signers: Vec<String>,
     pub confirmations_required: u64,
     pub confirmations: Vec<MultisigConfirmation>,
+    pub signature_info: Option<SignatureInfo>,
```

Comment:
`SignatureInfo` should not be an option and I would have all the fields of that struct directly in the `MultisigExecutionDetails` struct (see doc, as there optional fields are mentioned, everything else should not be optional)